### PR TITLE
facebook global outage 04th Oct 2021

### PR DESCRIPTION
**Pull request by wasylni**
Update about the October 4th outage

### CODE REVIEW

**Comment by DamianFekete** [comment]
Not a lot of info.
If I would compare this with https://www.thousandeyes.com/blog/facebook-outage-deep-dive/ ...

**Comment by wasylni** [comment]
> Not a lot of info. If I would compare this with a previous outage https://www.thousandeyes.com/blog/facebook-outage-deep-dive/ ...

Yes, I hope they will share some more details later...

File: README.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
- - **[Uncategorized](#uncategorized)**
- - **[Other lists of postmortems](#other-lists-of-postmortems)**
- - **[Analysis](#analysis)**
- - **[Contributors](#contributors)**
+- **[Config Errors](#config-errors)**
+- **[Hardware/Power Failures](#hardwarepower-failures)**
+- **[Conflicts](#conflicts)**
+- **[Time](#time)**
+- **[Uncategorized](#uncategorized)**
+- **[Other lists of postmortems](#other-lists-of-postmortems)**
```
Comment (icco) [inline README.md:10]: Please revert whitespace changes.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,8 @@
 
 [Etsy](https://codeascraft.com/2012/01/23/solr-bittorrent-index-replication/). Sending multicast traffic without properly configuring switches caused an Etsy global outage.
 
+[Facebook](https://engineering.fb.com/2021/10/04/networking-traffic/outage/). October 4th 2021 global outage - Configuration changes on the backbone routers.
```
Comment (icco) [inline README.md:28]: Change summary to a complete sentence.

Comment (wasylni) [reply README.md]: updated

Comment (smagellan) [reply README.md]: this: https://engineering.fb.com/2021/10/05/networking-traffic/outage-details/

Comment (wasylni) [reply README.md]: thx

**Review by icco — APPROVED** [review]

**Pull request merged**